### PR TITLE
form no longer accepts invalid or empty dates

### PR DESCRIPTION
**Pull request by mhn94**
Fixes Issue #1514 

Now, using iziToast, when a user enters a From date greater than a To date or submits the Flow Board form without a date(s), a warning message pops up to let the user know without submitting the form:

Examples:
1) User submits the form with 2019-10-01 to 2019-09-04 date range:
invalid-date-range-resolved-w-outline-size

2) User submits form without specifying a date:
empty-date-resolved-w-outline-size

### CODE REVIEW

File: interface/patient_tracker/patient_tracker.php [hunk 1]
```diff
@@ -143,29 +143,30 @@
     //  Include Bootstrap and DateTimePicker
   call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker"));
 ?>
-    <link rel='stylesheet' href='<?php echo $css_header ?>' type='text/css'>
-
+<link rel='stylesheet' href='<?php echo $css_header ?>' type='text/css'>
+<link rel="stylesheet" href="../../assets/css/iziModalToast/iziToast.min.css" type="text/css">
+<script type="text/javascript" src="../../assets/js/iziModalToast/iziToast.min.js" type="text/javascript"></script>
```
Comment (muarachmann) [inline interface/patient_tracker/patient_tracker.php]: You can call this in the `called_required_libraries()`

File: interface/patient_tracker/patient_tracker.php [hunk 2]
```diff
@@ -208,7 +206,41 @@ function topatient(newpid, enc) {
...
+                });
+                return false;
+            }
+            return true;
+        } else {
+            iziToast.warning({
+                title: 'Caution:',
+                message:"<?php echo xls('You must enter a From and To date.'); ?>",
+            });
+            return false;
```
Comment (muarachmann) [inline interface/patient_tracker/patient_tracker.php]: Nice function ;)

File: interface/patient_tracker/patient_tracker.php (hunk 1 above)
Comment (mhn94) [reply interface/patient_tracker/patient_tracker.php]: Good suggestion! I just made this change and pushed it.

File: interface/patient_tracker/patient_tracker.php [hunk 3]
```diff
@@ -141,11 +141,9 @@
 <title><?php echo xlt("Flow Board") ?></title>
 <?php
     //  Include Bootstrap and DateTimePicker
-  call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker"));
+  call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker", "iziModalToast"));
```
Comment (muarachmann) [inline interface/patient_tracker/patient_tracker.php]: LGTM

**Pull request merged**